content/rendering-with-batch/aws-fis/fis-resilience-create.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Creating the experiment"
+date: 2022-09-20T00:00:00Z
+weight: 153
+---
+
+You may recall the configuration of the AWS Batch job definition, which specified 3 attempts inside the structure `retryStrategy`. With this configuration in place, in case of job failure, AWS Batch will retry running that job twice more. This step will define an AWS FIS experiment to test how our architecture will react if any of the Spot instances are terminated.
+
+## Creating the experiment
+
+Run the following command to generate the AWS Fault Injection Simulator (AWS FIS) experiement template file. The FIS Experiemnt will create an AWS FIS experiment to send a interruption signal to up to 5 of the EC2 Spot instances. The instance selection is randomly chosen based on the presence of the tags that were set when creating the compute environment `type : Spot`. If the EC2 instance is running an AWS Batch job, the job will exit and retry after other jobs finish or more EC2 capacity is dynamically added in response to the interruption signal.
```

Comment:
Rearrange the sentences of this paragraph. First explain the experiment that you are configuring and after tell the user to run the command.